packs/windows-iad-sami-compliance.conf

```diff
@@ -0,0 +1,218 @@
+// inspired from https://github.com/iadgov/Splunk-Assessment-of-Mitigation-Implementations/raw/master/SAMI_Business_Logic.xlsx
+// default interval 1d=86400sec
+{
+  "platform": "windows",
+  "queries": {
+    // AE
+    "win_compliance_secureboot": {
+      "query" : "select * from registry where key like 'HKEY_LOCAL_MACHINE\\SYSTEM\\CurrentControlSet\\Control\\SecureBoot\\State\\UEFISecureBootEnabled'",
+      "interval": "86400",
+      "platform": "windows",
+      "snapshot": true,
+      "description" : "Secureboot state"
```

Comment:
This description is pretty terse, would you mind expanding this, and the others below? Perhaps "Whether Secure Boot is enabled on this system."?

Reply:
yes. correct.